cmd/yaegi/yaegi.go

```diff
@@ -112,6 +113,7 @@ func main() {
 	i := interp.New(interp.Options{GoPath: build.Default.GOPATH, BuildTags: strings.Split(tags, ",")})
 	i.Use(stdlib.Symbols)
 	i.Use(interp.Symbols)
+	i.Use(unsafe.Symbols)
```

Comment:
I would like to make this line conditional to a command line option `-unsafe`.

Reply:
great idea